packages/InteractiveProfilingTool-Core.package/IPTTreeModel.class/instance/currentSelection..st

```diff
@@ -0,0 +1,6 @@
+accessing
+currentSelection: anObject
```

Comment:
I think we can be more precise in the naming of the argument. What do you think of `aNode`?

Reply:
Agreed and changed.